### ISSUE TITLE
diffuse release tarbomb

### ISSUE THREAD

**Issue by gsfr** [scitran/scitran#12]
The release is currently delivered as a [tarbomb](http://en.wikipedia.org/wiki/Tar_(computing)#Tarbomb). That's bad. Let's add the top-level scitran directory to the tarball. And we should also use a virtualenv, instead of the current ```--target``` path hackery.

These could be our new installation instructions:

First, install the following dependencies:
- [Docker](https://docs.docker.com/installation)
- [Python](https://www.python.org)
- [Python virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/)

Now, install SciTran:
```
curl https://storage.googleapis.com/scitran-dist/release.tar | tar x
virtualenv scitran/virtualenv
source scitran/virtualenv/bin/activate
pip install -U pip setuptools
pip install docker-py requests sh toml
```

Finally, launch SciTran:
```
sudo scitran/scitran.py start
```

**Comment by kofalt** [comment]
That all sounds reasonable :+1: 

Should be trivial to accomplish by modifying the release script.
We should draft instructions (`notes/releasing.md` or similar) for why the file whitelist exists.
Plus upload instructions (currently, upload `release-v.tar`, then overwrite `release.tar`)

When that change is made, don't forget to modify the development instructions as well.
Specifically the portion that begins with

> Download the release as above, but you only need two folders

**Comment by kevlarkevin** [comment]
Resolved by #15.  no more tarbomb.
